redirect-rules/simple/rule.js

```diff
@@ -0,0 +1,36 @@
+function redirectToConsentForm (user, context, callback) {
+  var hasConsented = user.app_metadata && user.app_metadata.has_consented;
+
+  // redirect to consent form if user has not yet consented
+  if (!hasConsented && context.protocol !== 'redirect-callback') {
+    var auth0Domain = auth0.baseUrl.match(/([^:]*:\/\/)?([^\/]+\.[^\/]+)/)[2];
+
+    context.redirect = {
+      url: configuration.CONSENT_FORM_URL +
+        '&auth0_domain=' + encodeURIComponent(auth0Domain) +
```

Comment:
It goes back to the the other point about using the pre-provisioned webtask that already works. If we were asking that people create their own, it is also an opportunity to educate about the need to be explicit about the callback URL domain.